UITests/ViewMemoryLeakTests.swift

```diff
@@ -26,14 +26,18 @@ class ViewMemoryLeakTests: KIFTestCase, UITextFieldDelegate {
 
     func testAboutHomeDisposed() {
         // about:home is already active on startup; grab a reference to it.
-        let browserViewController = UIApplication.sharedApplication().keyWindow!.rootViewController!
+        let browserViewController = getTopViewController()
         weak var aboutHomeController = getChildViewController(browserViewController, childClass: "HomePanelViewController")
 
-        // Change the page to make about:home go away.
-        tester().tapViewWithAccessibilityIdentifier("url")
-        let url = "\(webRoot)/numberedPage.html?page=1"
-        tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url)\n")
-        tester().waitForWebViewElementWithAccessibilityLabel("Page 1")
+        // We have to use an autorelease pool here or else the weak reference to the about:home controller won't get cleared
+        // right away although the strong reference on the browser controller is set to nil.
+        autoreleasepool {
```

Comment:
Sorry, I am a bit confused. Which `webView` do you mean? There is no such variable in `testAboutHomeDisposed`.

I did some more digging and I believe this could be a bug with KIF. Weak variables do not seem to be cleared automatically when deriving from `KIFTestCase` unless one uses an explicit autorelease pool (see my bug report: https://github.com/kif-framework/KIF/issues/739).

In fact, I found that it's enough to wrap the following line into an autorelease pool to make the test pass.

```
autoreleasepool {
    tester().clearTextFromAndThenEnterTextIntoCurrentFirstResponder("\(url)\n")
}
```

This is in line with the suspicion above since entering an URL will kick off the loading and dismiss and nillify the home panel view controller. So this is where apparently we need to add an explicit autorelease pool in order to propagate the nillification.